doc/schemas/node_openapi.json

```diff
@@ -986,30 +986,6 @@
         }
       }
     },
-    "/node/code": {
```

Comment:
We need this to figure out what code versions are accepted in the CCF network. It's a sanity check prior to node startup to make sure we're starting the right version, and also if we get confused during a code upgrade, it's helpful to know exactly what version(s) we can join to the network.

This may not be the endpoint you want to expose for SEV-SNP since the attestation looks different, so if you want to take it out, you can leave it undocumented (maybe just undocumented in the Azure REST API but still documented here) but available, at least for supporting SGX.

Reply:
Same here, this data will still be accessible by these endpoints https://github.com/microsoft/CCF/blob/main/doc/schemas/gov_openapi.json#L1690-L1734